test_plans/oneapi/ext_oneapi_current_device.asciidoc

```diff
@@ -0,0 +1,44 @@
+:sectnums:
+:xrefstyle: short
+
+= Test plan for ext oneapi current device
+
+This is a test plan for the device selector as described in SYCL 2020 revision 9 specification.
+
+Estimated development time is one day.
+
+== Testing scope
+
+=== Backend coverage
+
+All the tests described below are not backend-specific and are performed for any SYCL backend.
+
+=== Device coverage
+
+All tests construct a test device for which conformance is assessed. All of the tests described below are performed only on the default device that
+can be selected on the CTS command line.
+
+== Tests
+
+=== Feature test macro
+
+All of the tests should use `#ifdef SYCL_EXT_ONEAPI_CURRENT_DEVICE` so they can be skipped
+if feature is not supported.
+
+=== Function `get_current_device`
+
+1. Positive case. This test can be performed by seting default device, calling from a host thread the function sycl::ext::oneapi::experimental::this_thread::get_current_device 
+and compare function output with the default device which was set earlier, check thread ID 
+
+2. Negative case 1. Implement the same as positive but call function from kernel(not host) code. Use XFAIL to detect compilation error
```

Comment:
We don't usually write tests in the CTS to detect compilation failures for illegal programs.  In fact, we should not do this in the CTS because there is no requirement for a SYCL implementation to diagnose a compilation error in this case.